1.60.1 fails to build on macOS

I get this error when trying to build:

```
launch-direct.c:881:5: error: use of undeclared identifier 'execvpe'
  881 |     execvpe (data->qemu, argv, env);        /* Run qemu. */
```

Elsewhere in the project, this is mentioned:

https://github.com/libguestfs/libguestfs/blob/6b592035130cde2501921f3861693ceb143591cb/lib/command.c#L621-L623

--- a/lib/launch-direct.c
+++ b/lib/launch-direct.c
@@ -878,7 +878,9 @@ launch_direct (guestfs_h *g, void *datav, const char *arg)
     if (g->pgroup)
       setpgid (0, 0);
 
-    execvpe (data->qemu, argv, env);        /* Run qemu. */
+    /* Assign to environ since execvpe is a GNU extension. */
+    environ = env;
+    execvp (data->qemu, argv);  /* Run qemu. */
     perror (data->qemu);
     _exit (EXIT_FAILURE);
   }

